aldryn_newsblog/models.py

```diff
@@ -137,7 +149,7 @@ class Article(TranslatedAutoSlugifyMixin,
     tags = TaggableManager(blank=True)
 
     related = SortedManyToManyField('self', verbose_name=_('related articles'),
-                                    blank=True)
+                                    blank=True, symmetrical=ARTICLE_RELATED_SYMMETRICAL)
```

Comment:
Well... how else can I be backwards compatible? And true... yes - the migration will occur or not depending on the setting.